examples/README.md

```diff
@@ -2,18 +2,36 @@
 
 These examples are hosted on this repository's GitHub pages.
 
+
+```
+# yellow tripdata
+wget https://d37ci6vzurychx.cloudfront.net/trip-data/yellow_tripdata_2023-01.parquet
+
+python3 -c "import pandas; pandas.read_parquet('yellow_tripdata_2023-01.parquet').to_json('yellow_tripdata_2023-01.jsonl', orient='records', lines=True)"
+```
+
```

Comment:
Didn't want to erase this as I'm cautious about erasing someone else's work, especially non-logic stuff.

But the example `index.html` explicitly specifies for the `green_tripdata` dataset.